wagtailio/project_styleguide/templates/patterns/pages/search_results/search_results.html

```diff
@@ -12,7 +12,10 @@ <h1 class="hero__heading">Search results</h1>
                         <label for="search" class="u-sr-only">Search</label>
                         <input id="search" type="search" placeholder="Enter your search term" class="form__input"
                             {% if search_query %} value="{{ search_query }}"{% endif %} aria-label="Search the site content"/>
-                        {% include "patterns/components/buttons/button.html" with arrow=True url=url title='Search' classes="button--dark" %}
+                        <button type="submit" class="button button--dark search__button" aria-label="Search the site content">
+                            <p class="button__text">Search</p>
+                            <svg class="arrow" aria-hidden="true"><use xlink:href="#arrow"></use></svg>
+                        </button>
```

Comment:
question: why not use the include?

Reply:
The button component is an **anchor** element, my thinking was that we need a **button** with `type="submit"`

https://github.com/wagtail/wagtail.org/blob/ee17ad61413d073a6659f610dbb8e23496caaa4d/wagtailio/project_styleguide/templates/patterns/components/buttons/button.html#L1-L6

Reply:
Aaah, I see. This doesn't feel good to me.

If it were me I would change `components/buttons/button.html` to be `components/buttons/button-link.html` and keep `components/buttons/button.html` to the semantic `<button>` element

this is one for the FE tech debt cc @William-Blackie